synapse/lib/layer.py

```diff
@@ -105,15 +105,39 @@ class LayerApi(s_cell.CellApi):
 
         self.layr = layr
         self.liftperm = ('layer:lift', self.layr.iden)
+        self.writeperm = ('layer:write', self.layr.iden)
 
     async def iterLayerNodeEdits(self):
         '''
         Scan the full layer and yield artificial nodeedit sets.
         '''
+
         await self._reqUserAllowed(self.liftperm)
         async for item in self.layr.iterLayerNodeEdits():
             yield item
 
+    async def storNodeEdits(self, nodeedits):
+
```

Comment:
For splice -> nodeedit migration case I do have meta data, but we would be re-generating it here.  What do you think about allowing meta to be passed in?

Reply:
Makes sense, I'll update it.